Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,299 @@
+    final have_fan=false,
+    nPorts_aChiWat=1,
+    nPorts_bChiWat=1);
+  // Mass flow rates
+  parameter Modelica.Units.SI.MassFlowRate mDis_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of district cooling side"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mBui_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of building cooling side"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mByp_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate through the bypass segment"
+    annotation(Dialog(group="Nominal condition"));
+  // Pressure drops
+  parameter Modelica.Units.SI.PressureDifference dpConVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop in the control valve"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.PressureDifference dpCheVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop in the check valve"
+    annotation(Dialog(group="Nominal condition"));
+  // Controller parameters
+  parameter Real k(
+    final min=0,
+    final unit="1")=1
+    "Gain of controller"
+    annotation (Dialog(group="PID controller"));
+  parameter Modelica.Units.SI.Time Ti(
+    final min=Modelica.Constants.small)=120
+    "Time constant of integrator block"
+    annotation (Dialog(group="PID controller",enable=controllerType == CDL.Types.SimpleController.PI or controllerType == CDL.Types.SimpleController.PID));
+  parameter Modelica.Units.SI.Time Td(
+    final min=0)=0.1
+    "Time constant of derivative block"
+    annotation (Dialog(group="PID controller",enable=controllerType == CDL.Types.SimpleController.PD or controllerType == CDL.Types.SimpleController.PID));
+  parameter Real yMax(
+    final start=1)=1
+    "Upper limit of output"
+    annotation (Dialog(group="PID controller"));
+  parameter Real yMin=0.01
+    "Lower limit of output"
+    annotation (Dialog(group="PID controller"));
+  parameter Modelica.Blocks.Types.SimpleController controllerType=Modelica.Blocks.Types.SimpleController.PI
+    "Type of controller"
+    annotation (Dialog(group="PID controller"));
+  // Advanced parameters
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial
+    "Type of energy balance: dynamic (3 initialization options) or steady state";
+  parameter Modelica.Units.SI.PressureDifference[3] dp_nominal=500*{1,-1,1}
+    "Nominal pressure drop in pipe junctions";
+  Modelica.Blocks.Interfaces.RealInput TSetDisRet
+    "Setpoint for the minimum district return temperature"
+    annotation (Placement(transformation(extent={{-338,-20},{-298,20}})));
+  Modelica.Blocks.Interfaces.RealOutput Q_flow(
+    final quantity="Power",
+    final unit="W",
+    displayUnit="kW")
+    "Measured power demand at the ETS"
```

Comment:
Better would be "Measured heating demand at the ETS". Also the quantity should be `HeatFlowRate`.